apps/backend/src/mcp/tools/device_management.py

```
@@ -26,6 +26,150 @@
+                raise ValidationError(
+                    message=f"Device with hostname '{hostname}' already exists",
+                    field="hostname",
+                    value=hostname
+                )
+                
+            else:
+                # Other error
+                try:
+                    error_data = response.json()
+                    error_message = error_data.get("detail", f"HTTP {response.status_code}")
+                except Exception as json_error:
```

Ruff (0.12.2)

Local variable `json_error` is assigned to but never used

Remove assignment to unused variable `json_error`

(F841)